dll/ime/msctfime/lang/tr-TR.rc

```diff
@@ -0,0 +1,23 @@
+/*
+ * PROJECT:     ReactOS msctfime.ime
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATOR:  Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
+ */
+
+LANGUAGE LANG_TURKISH, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "Tamam"
+    IDS_CANCEL           "İptal"
+    IDS_ABORT            "Durdur"
+    IDS_RETRY            "Yeniden Dene"
+    IDS_IGNORE           "Yoksay"
+    IDS_YES              "Evet"
+    IDS_NO               "Hayır"
```

Comment:
I've not asked to remove accelerators for these ones... see en-US

Reply:
> I've not asked to remove accelerators for these ones... see en-US

alr i fixed that see new commit 👍

Reply:
Resolved. Thanks.